src/store/storeConfig.ts

```diff
@@ -170,8 +171,18 @@ export const storeConfig: StoreOptions<State> = {
       meshInsightsFetching || serviceInsightsFetching || externalServicesFetching,
     getServiceResourcesFetching: ({ serviceInsightsFetching, externalServicesFetching }) =>
       serviceInsightsFetching || externalServicesFetching,
-    getChart: ({ overviewCharts }) => (chartName: string) => overviewCharts[chartName],
     getZonesInsightsFetching: ({ zonesInsightsFetching }) => zonesInsightsFetching,
+    getChart: (state) => {
+      return ({ chartName, title, subtitle = undefined, showTotal = false, isStatusChart = false }: { chartName: string, title: string, subtitle?: string, showTotal?: boolean, isStatusChart?: boolean }) => {
+        return {
```

Comment:
bit: Could this be made a little easier to read, either multiline or I noticed you also have a new type for what looks like this in `index.d.ts`, could that be used here?

Reply:
Right, I’ve changed this to use the type.